### DISCUSSION TITLE
Print Modal Only?

### DISCUSSION THREAD

**Discussion by MACscr** [wire-elements/modal#170] [Q&A]
Anyone figured out a way to print only the modal? I find tailwind to be a pain when trying to create printer friendly versions. In this case I am opening an invoice in the modal. Looks great until I try to print and it shows everything behind it as well and of course the content is centered on the page instead of the top. Anyone have any suggestions?

**Comment by gazzoy** [comment]
+1

**Comment by PhiloNL** [comment]
I would recommend adding a print button that opens a new tab with just the invoice. This makes it much easier to control exactly how the invoice is printed.
